crates/vfio-ioctls/Cargo.toml

```diff
@@ -14,6 +14,7 @@ mshv = ["mshv-ioctls", "mshv-bindings"]
 
 [dependencies]
 byteorder = ">=1.2.1"
+libc = "0.2.108"
```

Comment:
Can't we relax this dependency?

Reply:
libc = ">=0.2.39" should work most of our cases.